corehq/apps/userreports/tests/test_report_builder.py

```diff
@@ -171,13 +173,12 @@ def test_reference_for_forms(self):
         )
         # todo: we should filter out some of these columns
         expected_property_names = [
-            "doc_id", "inserted_at", "completed_time", "started_time", "username", "userID", "@xmlns", "@name",
-            "App Version", "deviceID", "location", "app_id", "build_id", "@version", "state", "last_sync_token",
-            "partial_submission", "received_on", "edited_on", "submit_ip",
+            "doc_id", "inserted_at", "completed_time", "started_time", "username", "userID", "@xmlns",
+            "@name", "App Version", "deviceID", "location", "app_id", "build_id", "@version", "state",
+            "last_sync_token", "partial_submission", "received_on", "edited_on", "submit_ip",
             "form.first_name", "form.last_name", "form.children", "form.dob", "form.state",
-            "form.case.@date_modified", 'form.case.@user_id', 'form.case.@case_id', 'form.case.update.first_name',
-            'form.case.update.last_name', "count", "hq_user",
-        ]
+            "form.case.@date_modified", 'form.case.@user_id', 'form.case.@case_id',
+            'form.case.update.first_name', 'form.case.update.last_name', "count", "hq_user", ]
```

Comment:
Did anything change in this block, or is it just the linebreaks?

Reply:
This code change was removed, marking resolved.